content/About Arduino/Trademarks, Logos, and Licenses/Arduino-and-Community-logos.md

```diff
@@ -16,8 +16,8 @@ The Arduino logo can be found in the following colors:
 
 The Arduino logo is a registered trademark and cannot be used by third parties without the trademark owner’s authorization. Generally speaking, you **cannot** use the Arduino Logo (with or without the word *Arduino*) to identify or advertise your products and/or services. This includes:
 
-* Compatible products <!-- link to Compatible products section-->
-* Publications <!-- link to Courses, workshops and publications section-->
+* [Compatible products](https://support.arduino.cc/hc/en-us/sections/360004749260-Compatible-Products)
+* [Publications](https://support.arduino.cc/hc/en-us/sections/360005977760-Courses-Workshops-and-Publications)
```

Comment:
```suggestion
* [Courses, Workshops, and Publications](https://support.arduino.cc/hc/en-us/sections/360005977760-Courses-Workshops-and-Publications)
```
Does it not apply to all three?

@maria-tonini

Reply:
yes it does. Thank you for spotting the inconsistency in the older text.